applications/geoclaw/bowl_slosh/regressions.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 # absolute path to application we are testing
-application=$FCLAW_APPLICATIONS_BUILD_DIR/geoclaw/bowl_slosh/slosh
+application=$FCLAW_APPLICATIONS_BUILD_DIR/geoclaw/bowl_slosh/bowl_slosh
 
 # change to source dir for working directory
-cd $FCLAW_APPLICATIONS_SRC_DIR/geoclaw/bowl_slosh/regression
+cd $FCLAW_APPLICATIONS_SRC_DIR/geoclaw/bowl_slosh/
 
 # run programs, exit script with nonzero on failure (or else script will exit with value of last program run)
-$FCLAW_MPIRUN $FCLAW_MPI_TEST_FLAGS $application -F regression.ini || exit 1
+$FCLAW_MPIRUN $FCLAW_MPI_TEST_FLAGS $application -F regression.ini --user:claw-version=4 || exit 1
+$FCLAW_MPIRUN $FCLAW_MPI_TEST_FLAGS $application -F regression.ini --user:claw-version=5 || exit 1
```

Comment:
There is no `user:claw-version` option declared for this example, so specifying this option will fail.